irc/pounce/Portfile

```diff
@@ -35,18 +39,23 @@ checksums           rmd160  1ae7f9fe9239352992c2ad67107e13a44d4ef5b2 \
                     sha256  97f245556b1cc940553fca18f4d7d82692e6c11a30f612415e5e391e5d96604e \
                     size    50801
 
+# Modern explicit_bzero uses memset_s(), which is not
+# available on macOS <10.9, so emulate it with
+# volatile memset().
+if {${os.platform} eq "darwin" && ${os.major} < 13} {
+    patchfiles          patch-explicit-bzero.diff
+}
+
 depends_build-append \
                     port:pkgconfig
 
 depends_lib-append  port:curl \
                     port:libretls \
                     port:sqlite3
 
-build.pre_args-append \
-                    MANDIR=${prefix}/share/man
+compiler.c_standard 2011
 
-destroot.pre_args-append \
-                    MANDIR=${prefix}/share/man
```

Comment:
did you verify that when removing this the man pages still get installed in the correct place?